kolibri/plugins/document_pdf_render/assets/src/views/index.vue

```diff
@@ -1,47 +1,64 @@
 <template>
 
-  <div ref="container" class="container" allowfullscreen>
```

Comment:
Something seems to have gone very awry here - the pdf renderer index.vue appears to be an exact copy of the HTML5 renderer index.vue now - is this intentional?